plugin/src/gatsby-node.ts

```diff
@@ -20,6 +20,10 @@ export function pluginOptionsSchema({ Joi }: PluginOptionsSchemaArgs) {
     storeUrl: Joi.string().required(),
     downloadImages: Joi.boolean(),
     verboseLogging: Joi.boolean(),
+    typePrefix: Joi.string()
+      .pattern(new RegExp('(^[A-Z][\w|\d]*)'))
```

Comment:
Does `\w` already cover what `\d` would match? I think `\w` includes letters and numbers as well as underscores.

Reply:
You're correct! I was thinking `w` was characters and `d` was digits, but `w` is words (including digits). Thanks! Fixed!